package.json

```diff
@@ -4,31 +4,75 @@
   "version": "2.0.0",
   "description": "Status Panel for Grafana",
   "scripts": {
-    "build": "grafana-toolkit plugin:build",
-    "test": "grafana-toolkit plugin:test",
-    "dev": "grafana-toolkit plugin:dev",
-    "watch": "grafana-toolkit plugin:dev --watch",
-    "sign": "grafana-toolkit plugin:sign",
-    "serve": "/run.sh",
-    "start": "concurrently --kill-others \"yarn watch\" \"yarn serve\""
-  },  
+    "build": "webpack -c ./.config/webpack/webpack.config.ts --env production",
+    "dev": "webpack -w -c ./.config/webpack/webpack.config.ts --env development",
+    "e2e": "yarn exec cypress install && yarn exec grafana-e2e run",
+    "e2e:update": "yarn exec cypress install && yarn exec grafana-e2e run --update-screenshots",
```

Comment:
should this still be referencing cypress?

Reply:
yeah good point, it was scaffolded before we moved to playwright, I can probably update this.

Reply:
Moved to latest and playwright, no more cypress